packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -543,13 +543,16 @@ class ScrollingFlameChartRowState<V> extends State<ScrollingFlameChartRow>
     while (min < max) {
       final mid = min + ((max - min) >> 1);
       final node = nodes[mid];
-      if (x >= node.rect.left && x <= node.rect.right) {
+      final zoomedNodeRect = node.selectable
```

Comment:
maybe make this a helper method on the node that returns the zoomedRect and just returns the raw rect when then node is not selectable.
it isn't obvious to me at a glance why selectable nodes should use one coordinate system while non selectable ones should use another.

Reply:
done. added a comment to clarify.